app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,46 @@
+class QueueItemsController < ApplicationController
+  before_action :require_user
+  
+  def index
+    @queue_items = current_user.queue_items
+  end
+  
+  def create
+    video = Video.find(params[:video_id])
+    queue_the_item(video)
+    redirect_to my_queue_path
+  end
+  
+  def destroy
+    queued_item = QueueItem.find(params[:id])
+    queued_item.destroy if belongs_to_current_user?(queued_item)
+    
+    respond_to do |format|
```

Comment:
By default Rails will server the html format so you can actually do away with the `respond_to` block.